sass/includes/_record-matters.scss

```diff
@@ -63,13 +63,14 @@
     &__buttons {
         margin-top: 1rem;
         text-align: center;
-        display: grid;
+        display: flex;
         grid-template-columns: 1fr 1fr 1fr;
         grid-gap: 1.5rem;
 
         @media (max-width: $screen__md) {
             grid-template-columns: 1fr;
             grid-gap: 0;
```

Comment:
Again, we can remove these rules now

Reply:
I think we need to keep `grid-gap: 1.5rem;` in, as the buttons will come together without it. Unless there's a way that we can add standardised margins to buttons?